### PR TITLE
add prometheus additional scrape config for helm installation and typ…

### PR DESCRIPTION
**Pull request by federeynimbux**
…o metrics server typo

### CODE REVIEW

**Review by simple-me — DISMISSED** [review]

File: helm.tf [hunk 1]
```diff
@@ -154,7 +154,7 @@ resource "helm_release" "metrics_server" {
 
   set {
     name = "priorityClassName"
-    value = var.metrics_autoscaler_priorityclass
```
Comment (qemanuel) [inline helm.tf:157]: @federeynimbux why is this change for?

File: helm.tf [hunk 2]
```diff
@@ -154,7 +154,7 @@ resource "helm_release" "metrics_server" {
...
+    value = var.metrics_server_priorityclass
```
Comment (qemanuel) [inline helm.tf:157]: @federeynimbux i think this mismatch the name in README (it is written like metrics_server_priorityClassName)

File: variables.tf [hunk 3]
```diff
@@ -564,6 +564,10 @@ variable "prometheus_metrics_retention" {
     default = "14d"
 }
 
+variable "prometheus_additionalscrapeconfigs" {
```
[the remark this answers is not in the document]
Comment (qemanuel) [reply variables.tf:567]: @federeynimbux The updates in the CHANGELOG are missing

**Review by qemanuel — CHANGES_REQUESTED** [review]

File: helm.tf (hunk 2 above)
Comment (federeynimbux) [reply helm.tf]: changes were made

File: variables.tf (hunk 3 above)
[the remark this answers is not in the document]
Comment (federeynimbux) [reply variables.tf]: changes were made and added changelog

**Review by qemanuel — APPROVED** [review]
LGTM

**Pull request merged**